app/assets/stylesheets/course/enrol_requests.css.scss

```diff
@@ -0,0 +1,43 @@
+#enrol_request_form {
```

Comment:
Avoid using id selectors<br>Selector `enrol_request_form` should be written in lowercase with hyphens